### DISCUSSION TITLE
Build a Blitz Full-Stack with multi Front-End

### DISCUSSION THREAD

**Discussion by NurdinDev** [blitz-js/blitz#2259] [Q&A] [answered]
There are 3 main parts for any Full-stack application The Database, Admin Dashboard, and The FrontEnd, and maybe just 2 parts Database, the admin dashboard, and FrontEnd on the same domain.
making the admin dashboard separate from your frontend has more advantages, are you agree with me?

I wondering if there is a way to build a single full-stack Blitzjs application for multi frontend for example (web and admin dashboard) so you can publish each front-end to a different domain.

I hope that makes sense.

**Comment by NurdinDev** [comment]
I guess this is the appropriate answer https://blitzjs.com/docs/tradeoffs#multi-client-api

**Comment by flybayer** [comment, marked as answer]
Currently that's not possible, but it's something we can explore in the future
